support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/subheading.tsx

```diff
@@ -154,14 +131,19 @@ function Subheading({
 			{subheadingCopy}
 			{!isGuardianAdLite && !isPending && !observerPrint && (
 				<>
-					<MarketingCopy
-						contributionType={contributionType}
-						isTier3={isTier3}
-						isPaper={isPaper}
-					/>
+					<span>
```

Comment:
Interested in why this was inlined - would it make more sense as a separate component (as before)?

Reply:
I inlined as its only called once, perhaps its a trade-off versus readability, happy to go with the later anyway. Updated to original component.